### DISCUSSION TITLE
Logout button takes me to Microsoft sign-off page

### DISCUSSION THREAD

**Discussion by live4live4** [provectus/kafka-ui#3547] [Q&A] [answered]
I got SSO working using Azure AD, config as follows:

`FROM provectuslabs/kafka-ui:v0.6.0

ENV KAFKA_CLUSTERS_0_PROPERTIES_SECURITY_PROTOCOL "SASL_SSL"
ENV KAFKA_CLUSTERS_0_PROPERTIES_SASL_MECHANISM "AWS_MSK_IAM"
ENV KAFKA_CLUSTERS_0_PROPERTIES_SASL_CLIENT_CALLBACK_HANDLER_CLASS "software.amazon.msk.auth.iam.IAMClientCallbackHandler"
ENV KAFKA_CLUSTERS_0_PROPERTIES_SASL_JAAS_CONFIG 'software.amazon.msk.auth.iam.IAMLoginModule required;'
ENV AUTH_TYPE 'OAUTH2'
ENV SERVER_FORWARDHEADERSSTRATEGY 'native'

ENV AUTH_OAUTH2_CLIENT_AUTH0_provider 'auth0'
ENV AUTH_OAUTH2_CLIENT_AUTH0_CLIENTID 'xxx'
ENV AUTH_OAUTH2_CLIENT_AUTH0_CLIENTSECRET 'xxx'
ENV AUTH_OAUTH2_CLIENT_AUTH0_SCOPE 'openid'
ENV AUTH_OAUTH2_CLIENT_AUTH0_ISSUER-URI 'https://login.microsoftonline.com/xxx/v2.0'
ENV AUTH_OAUTH2_CLIENT_AUTH0_USER_INFO_URI 'https://graph.microsoft.com//oidc/userinfo'
ENV AUTH_OAUTH2_CLIENT_AUTH0_USER_NAME_ATTRIBUTE 'email'`

When I try to logout https://kafka-ui.xxx.com/logout, it took me to Microsoft sigh-off page and it eventually will log out my whole Microsoft login session. 
<img width="380" alt="image" src="https://user-images.githubusercontent.com/46739143/227421092-2b436091-0c1b-420a-8f22-16e4f734ff24.png">

Ideally the log-out button only clears the user's session data inside Kafka UI itself. 

I'm not sure if it's some more config needs to be done or it's not a feature yet.

**Comment by Haarolean** [comment, marked as answer]
Hi, I believe I can't help you here, as this is a default implementation per OAuth2 specifications, we're just using the logout URL provided to us in one of the requests. You might wanna look into azure docs, there might be a way to tune it.
